src/build/cargo.rs

```diff
@@ -405,7 +389,7 @@ impl Executor for RlsExecutor {
                 _ => {}
             }
         } else {
-            cmd.status().expect("Couldn't execute rustc");
+            cmd.exec()?;
```

Comment:
This changes the semantics, I believe - it will still error out when the process can't be run, but *also* when the exit code is non 0 (e.g. dep can't be compiled). Maybe this will somehow prevent the spontaneous fingerprint crashes (as per my https://github.com/rust-lang-nursery/rls/issues/443#issuecomment-323648389)?